ecosystem.config.js

```diff
@@ -0,0 +1,22 @@
+/*global __dirname*/
+const path = require('path');
+
+module.exports = {
+  apps : [{
+    name: path.basename(__dirname),
+    script: 'lib/app.js',
+
+    // Options reference: https://pm2.io/doc/en/runtime/reference/ecosystem-file/
+    args: '--port 3002',
+    instances: 1,
```

Comment:
Oh, I get it, this is dev server mode.

Reply:
The main reason is because on Windows, at least, pm2 in cluster mode (more than 1 instance) opens a console window per instance. Having 8 consoles open was annoying.